package.json

```diff
@@ -137,6 +142,40 @@
 					"when": "spectral:hasSpectralInstalled"
 				}
 			]
+		},
+		"configuration": {
+			"type": "object",
+			"title": "Spectral",
+			"properties": {
+					"spectral.scan.engines.useSecretsEngine": {
+							"type": "boolean",
+							"default": false,
+							"description": "Use Secrets and misconfigurations engine kind (Supported with Spectral version higher then 1.10.48)",
```

Comment:
Secrets and misconfigurations engine (Supported from Spectral version 1.10.48 and above)

remove the `kind`, same for all others

Reply:
fixed, https://github.com/SpectralOps/vscode-extension/pull/7/commits/e1fded33e247a3fd2e4a7e247f1b8c1868623af6